src/transformers/__init__.py

```diff
@@ -140,6 +140,7 @@
         "is_tensorboard_available",
         "is_wandb_available",
     ],
+    "integrations.executorch": [],
```

Comment:
Remove this one -- with this import present, any `transformers` import would attempt to load the contents of `integrations/executorch.py`, which require `torch`